docs/proposals/versioning.md

```diff
@@ -0,0 +1,25 @@
+## Plan for versioning APBs
+
+### Current status:
+* Currently all APBs are 0.1.0
+  * This is stored in a label on the image titled “com.redhat.apb.version”
+* Plan is to bump all APBs to a newer version to coincide with 3.7 Broker
+
+### Questions:
+* Should this be a major version bump? I.e. 1.0.0, or minor would be 0.2.0
```

Comment:
+1 to doing a major version jump right now to something like 1.0.0.  A
minor version jump is not appropriate because minor versions should be
compatible with previous minor versions. That is they should not be a
breaking change.